### ISSUE TITLE
Feature request: Additional Tooltip Info

### ISSUE THREAD

**Issue by TheKrowi** [TheKrowi/Krowi_AchievementFilter#59]
Add additional tooltip info for achievements like Pest Control to see which pests still to kill.

**Comment by TheKrowi** [comment]
**Achievements done:**
- To All The Squirrels I've Loved Before
- To All The Squirrels Who Shared My Life
- To All the Squirrels Who Cared for Me
- To All the Squirrels I Once Caressed?
- To All the Squirrels Through Time and Space
- To All the Squirrels I Love Despite Their Scars
- To All the Squirrels I Set Sail to See
- To All the Squirrels I've Loved and Lost
- To All the Squirrels Hidden Till Now
- Pest Control

**Comment by TheKrowi** [comment]
**Achievements to do:**
-

**Comment by TheKrowi** [comment]
WorldExplorer = 46,		-- "World Explorer"
	-- LoveCritters = 1206,	-- "To All The Squirrels I've Loved Before"
	-- LoveCritters2 = 2557,	-- "To All The Squirrels Who Shared My Life"
	-- LoveCritters3 = 5548,	-- "To All the Squirrels Who Cared for Me"
	-- LoveCritters4 = 6350,	-- "To All the Squirrels I Once Caressed?"
	-- LoveCritters5 = 14728,	-- "To All the Squirrels Through Time and Space"
	-- LoveCritters6 = 14729,	-- "To All the Squirrels I Love Despite Their Scars"
	-- LoveCritters7 = 14730,	-- "To All the Squirrels I Set Sail to See"
	-- LoveCritters8 = 14731,	-- "To All the Squirrels I've Love and Lost"
	-- PestControl = 2556,		-- "Pest Control"
	WellRead = 1244,		-- "Well Read"
	HigherLearning = 1956,	-- "Higher Learning"
	EatYourGreens = 12588,	-- "Eat Your Greens"

	TastesLikeChicken = 1832,	-- "Takes Like Chicken"
	HappyHour = 1833,		-- "It's Happy Hour Somewhere"
	CataclysmicallyDelicious = 5753,-- "Cataclysmically Delicious"
	DrownYourSorrows = 5754,	-- "Drown Your Sorrows"
	PandarenCuisine = 7329,		-- "Pandaren Cuisine"
	PandarenDelicacies = 7330,	-- "Pandaren Delicacies"
	DraenorCuisine = 9502,		-- "Draenor Cuisine"
	BrewfestDiet = 1185,		-- "The Brewfest Diet"
	DarkmoonFaireFeast = 6026,	-- "Fairegoer's Feast"

	RightAsRain = 5779,		-- "You'll Feel Right as Rain"

	Scavenger = 1257,			-- "The Scavenger"
	OutlandAngler = 1225,		-- "Outland Angler"
	NorthrendAngler = 1517,		-- "Northrend Angler"
	Limnologist = 5478,			-- "The Limnologist"
	Oceanographer = 5479,		-- "The Oceanographer"
	PandarianAngler = 7611,		-- "Pandarian Angler"
	DraenorAngler = 9462,		-- "Draenor Angler"

	GourmetOutland = 1800,		-- "The Outland Gourmet"
	GourmetNorthrend = 1779,	-- "The Northrend Gourmet" (last part)
	GourmetCataclysm = 5473,	-- "The Cataclysmic Gourmet" (last part)
	GourmetPandaren = 7327,		-- "The Pandaren Gourmet" (last part)
	--GourmetWinter = 1688,		-- "The Winter Veil Gourmet" -- requires proper season; waiting on season detection feature?
	GourmetDraenor = 9501,		-- "The Draenor Gourmet"
	LegionMenu = 10762,			-- "The Legion Menu"
	BattleMenu = IsAlliance and 12744 or 12746,		-- "The Kul Tiran Menu" / "The Zandalari Menu"

	ThreeSheetsToTheWind = 13061,	-- "Three Sheets to the Wind"

	MediumRare = 1311,			-- "Medium Rare"
	BloodyRare = 1312,			-- "Bloody Rare"
	NorthernExposure = 2256,	-- "Northern Exposure"
	Frostbitten = 2257,			-- "Frostbitten"
	Glorious = 7439,			-- "Glorious!"
	BattleMonsters = 12587,		-- "Unbound Monstrosities"

	StoodInTheFire = 5518,		-- "Stood in the Fire"
	SurveyingTheDamage = 4827,	-- "Surveying the Damage"
	WhaleShark = 4975,			-- "From Hell's Heart I Stab at Thee"

	LetItSnow = 1687,		-- "Let It Snow"
	FistfulOfLove = 1699,	-- "Fistful of Love"
	BunnyMaker = 2422,		-- "Shake Your Bunny-Maker"
	CheckYourHead = 291,	-- "Check Your Head"
	TurkeyLurkey = 3559,	-- "Turkey Lurkey"

	MissionBarrens = IsAlliance and 12896 or 12867, -- Azeroth at War: The Barrens
	MissionLordaeron = IsAlliance and 12898 or 12869, -- Azeroth at War: After Lordaeron
	MissionKalimdor = IsAlliance and 12899 or 12870, -- Azeroth at War: Kalimdor on Fire

	--WhatWeRideInTheShadows = 14770,	-- "What We Ride in the Shadows"
	AfterlifeExpress = 14771,				-- "The Afterlife Express"

	-- Statistics:
	Stat_ConsumeDrinks = 346,	-- "Beverages consumed"
	Stat_ConsumeFood = 347,		-- "Food eaten"
	--1774 "Different beverages consumed"
	--1775 "Different foods eaten"

**Comment by Rezy942** [comment]
Not sure if already done but i would love this feature!
Overachiever had it and it was amazing.

**Comment by TheKrowi** [comment]
This is a work in progress still. This comment lists all the ones that are done: https://github.com/TheKrowi/Krowi_AchievementFilter/issues/59#issuecomment-1302468096
